query_refinery_health.tf

```diff
@@ -1,3 +1,46 @@
+resource "honeycombio_column" "collect_cache_buffer_overrun" {
```

Comment:
loop over a list instead, more readable and avoid a lot of duplication and simplify depends_on  :)